serveradmin/servershell/views.py

```diff
@@ -165,12 +160,12 @@ def get_results(request):
 def export(request):
     term = request.GET.get('term', '')
     try:
-        query_args = parse_query(term, filter_classes)
-        q = Query(query_args).restrict('hostname')
+        query_args = parse_query(term)
+        query = Query(query_args).restrict('hostname')
     except (QueryError, ValidationError) as error:
         return HttpResponse(str(error), status=400)
 
-    hostnames = u' '.join(server['hostname'] for server in q)
+    hostnames = u' '.join(server['hostname'] for server in query)
```

Comment:
`unicode` string should not be necessary, right?

Reply:
No, we are are running Python 3 anyway.  I am removing it.